task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,19 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (!(num % 3) && !(num % 5)) {
+        return 'FizzBuzz';
+    } else {
+        if (!(num % 5)) {
+            return 'Buzz';
+        } else {
+            if (!(num % 3)) {
+                return 'Fizz';
+            } else {
+                return num;
+            }
+        }
+    }
```

Comment:
Better, but still not very readable. You've create a pyramid of doom. Since you use return statements in if block you can omit else blocks and flatten the structure 

`if (!(num % 3) && !(num % 5)) {
        return 'FizzBuzz';
 }

if (!(num % 5)) {
            return 'Buzz';
} 
 if (!(num % 3)) {
            return 'Fizz';
}
           
return num;
`

![image](https://user-images.githubusercontent.com/7272208/87551971-bf5cc980-c6b9-11ea-84c6-315428851eef.png)